entity-framework/core/miscellaneous/platforms.md

```diff
@@ -14,25 +14,31 @@ We want EF Core to be available to developers on all modern .NET implementations
 
 The following table provides guidance for each .NET implementation:
 
-| EF Core                       | 3.1        | 5.0             |
-|:------------------------------|:-----------|:----------------|
-| .NET Standard                 | 2.0        | 2.1             |
-| .NET Core                     | 2.0        | 3.0             |
-| .NET Framework<sup>(1)</sup>  | 4.7.2      | (not supported) |
-| Mono                          | 5.4        | 6.4             |
-| Xamarin.iOS<sup>(2)</sup>     | 10.14      | 12.16           |
-| Xamarin.Mac<sup>(2)</sup>     | 3.8        | 5.16            |
-| Xamarin.Android<sup>(2)</sup> | 8.0        | 10.0            |
-| UWP<sup>(3)</sup>             | 10.0.16299 | TBD             |
-| Unity<sup>(4)</sup>           | 2018.1     | TBD             |
+EF Core                       | 3.1        | 5.0    | 6.0
+----------------------------- | ---------- | ------ | ---
+.NET                          |            | 5.0    | 6.0
+.NET Core                     | 3.1        |        |
+.NET Standard<sup>(1)</sup>   | 2.0        | 2.1    |
+.NET Framework<sup>(2)</sup>  | 4.7.2      |        |
+Mono                          | 5.4        | 6.4    |
```

Comment:
I honestly have no idea what the state of "classic" Mono is. Are they going to do another release? Are all the Mono-specific APIs like GTK# compatible with (or included in) .NET 6? I couldn't find much information about it...

Reply:
I'd say, who cares any more...